tests/api_connexion/endpoints/test_task_instance_endpoint.py

```diff
@@ -219,6 +224,50 @@ def test_should_respond_200(self, username, session):
             "triggerer_job": None,
         }
 
+    @parameterized.expand(
+        [
+            ("Exception serialization", _BadExecutorConfig(), "{}"),
```

Comment:
```suggestion
            ("Exception serialization", BadExecutorConfig(), "{}"),
```
just cus ... it's a test so it doesn't matter